source/settings/core.rst

```diff
@@ -2934,11 +2927,6 @@ See :ref:`settings` for list of all setting groups.
      :dovecot_core:ref:`auth_allow_cleartext` in that even non-cleartext
      authentication mechanisms aren't allowed without SSL/TLS.
 
-     Note that SSL is still not required for "secured" connections:
```

Comment:
This should be kept in some way, e.g.:

      Note that SSL is still not required for "secured" connections (client IP is local IP, i.e. localhost/interface IP).